moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -52,14 +78,36 @@ export const useStyles = makeStyles({
     height: 50,
     position: "absolute",
     alignItems: "center",
-    right: 32,
+    width: "21rem",
+    left: "1rem",
+    top: ".5rem",
+    zIndex: 2,
     // Keep geocoder input in set position when mapbox-gl-geocoder.css media queries kick in
     "@media (max-width:640px)": {
       top: 32,
     },
   },
+  exampleWrapper: {
```

Comment:
this seems like copy/pasta, is there a more descriptive name than example?

Reply:
It's a total copy pasta. I don't think it's being used anywhere in the code.